dm/pkg/conn/baseconn.go

```diff
@@ -259,11 +275,20 @@ func (conn *BaseConn) ApplyRetryStrategy(tctx *tcontext.Context, params retry.Pa
 	return conn.RetryStrategy.Apply(tctx, params, operateFn)
 }
 
-// close should not be used by functions other than BaseDB.CloseBaseConn.
+// close returns the connection to the connection pool, has the same meaning of sql.Conn.Close
```

Comment:
this one has the same meaning of `sql.Conn.Close()`, i.e. put into connection pool